src/overlay/FlowControl.h

```diff
@@ -167,11 +167,8 @@ class FlowControl
     // This method return last timestamp (if any) when peer had no available
     // outbound capacity (useful to diagnose if the connection is stuck for any
```

Comment:
Nit: Update comment for new return type.

Reply:
thanks, it's fixed now.